src/docs/patterns/lists.mdx

```diff
@@ -1,56 +1,193 @@
 
-<!-- KEYWORDS: design pattern list status sort table drag drop action -->
+import ListBannerImage from '../../app/assets/design-patterns/lists/lists.svg';
 
-# List Patterns
+<!-- Common Use Cases -->
 
-Lists are frequently used throughout PX Blue applications. Typically, lists are used to represent groups of similar data. They can be used to show simple name-value pairs in a sidebar all the way to complex timelines or device lists.
+import NotificationListDesktopImage from '../../app/assets/design-patterns/lists/notification-list-desktop.png';
+import NotificationListMobileImage from '../../app/assets/design-patterns/lists/notification-list-mobile.png';
+import DashboardListDesktopImage from '../../app/assets/design-patterns/lists/dashboard-list-desktop.png';
+import DashboardListMobileImage from '../../app/assets/design-patterns/lists/dashboard-list-mobile.png';
+import AnatomyImage from '../../app/assets/design-patterns/lists/anatomy.png';
+
+<!-- Variations -->
+
+import SimpleListImage from '../../app/assets/design-patterns/lists/simple-list.png';
+import StatusListImage from '../../app/assets/design-patterns/lists/status-list.png';
+import ScorecardImage from '../../app/assets/design-patterns/lists/scorecard.png';
+import ContactsImage from '../../app/assets/design-patterns/lists/contacts.png';
+import IconAlignImage from '../../app/assets/design-patterns/lists/icon-align.png';
+import NotificationBucketsImage1 from '../../app/assets/design-patterns/lists/notification-buckets-1.png';
+import NotificationBucketsImage2 from '../../app/assets/design-patterns/lists/notification-buckets-2.png';
+
+<!-- Behaviors -->
+
+import GlobalActionAppBarImage from '../../app/assets/design-patterns/lists/action-button-in-appbar.png';
+import GlobalActionPanelHeaderImage from '../../app/assets/design-patterns/lists/action-button-in-panel-header.png';
+import GlobalActionAsButtonImage from '../../app/assets/design-patterns/lists/action-button-as-button.png';
+import SortableListImage from '../../app/assets/design-patterns/lists/sortable-list.gif';
+import LocalListActionsImage from '../../app/assets/design-patterns/lists/local-list-actions.png';
+import MultiSelectImage from '../../app/assets/design-patterns/lists/multi-select-list.png';
+import NavigateListImage from '../../app/assets/design-patterns/lists/navigate-list.png';
+
+<!-- Specs -->
+
+import Spec1Image from '../../app/assets/design-patterns/lists/spec-1.png';
+import Spec2Image from '../../app/assets/design-patterns/lists/spec-2.png';
+import Spec3Image from '../../app/assets/design-patterns/lists/spec-3.png';
+import Spec4Image from '../../app/assets/design-patterns/lists/spec-4.png';
+import Spec5Image from '../../app/assets/design-patterns/lists/spec-5.png';
+import SpecTypographyImage from '../../app/assets/design-patterns/lists/spec-typography.png';
+
+<!-- KEYWORDS: design pattern list status sort table drag drop action data -->
+
+# Lists & Tables
```

Comment:
Personally, lists and tables are just two sides of one coin. Lists focus more on individual items (you read them by row), while tables focus more on trends (you read them by column). The difference between the two is blurred — how many components can I add to a list before it turns into a table? Does a list with some bare table headers suddenly become a table?

Reply:
I agree that the distinction between the two is muddled. 

I don't necessarily agree that a table is _read_ by column or that it is meant to focus on trends — You read a table holistically (if you just look at a column it means nothing without knowing what the row is, and often you're looking for correlations between columns) and if you are interested in trends, you wouldn't try to find them in raw data, you'd use a chart / graph. And I don't think quantity of items is a deciding factor — you can have a list with 10k items and a table with 3 rows and 2 columns. It's more about the structure of the data. 

I do think you're on the right track though...when you're thinking about 'columns', that's when you're getting into table territory and when you're thinking more about the 'item' it's more of a list. 

I am struggling to come up with a definition of when to use one versus the other. It's kind of a 'know it when you see it' kind of thing.

If we're going to discuss tables in this pattern though, then we need to be clear on what the difference is.

My biggest qualm is that List components are not well suited for building tables (because there are no columns), so I would not want to encourage people to use them for 'the same things.'

Reply:
Maybe the distinction can be around whether you need column labels or not. If it's not clear what the data is without headers, then it's a table. If you can understand the entire item without labels, it's a list?

Reply:
Discussion during the OH — go with the status quo, log an issue for it in the backlog to talk to studio blue about it